src/constants/networks.js

```diff
@@ -213,6 +213,13 @@ export function findNetworkByChainId(chainId, crossToNetworkId) {
   )
 }
 
+export function findNetworkById(id) {
```

Comment:
what's the difference between this function and the existing one? I think the naming is the only difference, functionally?

Reply:
Removed